packages/gatsby-cli/src/structured-errors/error-schema.ts

```diff
@@ -20,6 +20,7 @@ export const errorSchema: Joi.ObjectSchema<IStructuredError> = Joi.object().keys
         })
       )
       .allow(null),
+    errorCategory: Joi.string().valid([`USER`, `SYSTEM`]),
```

Comment:
it's already part of an error, should we prefix it with error?
```suggestion
    category: Joi.string().valid([`USER`, `SYSTEM`]),
```